### PR TITLE
improve docstring of SparseMatrixCSC

### PR DESCRIPTION
**Pull request by abraunst**
Give the poor user a hint about constructing these matrices by cross-referencing to `spzeros` and `sparse`.

### CODE REVIEW

**Comment by andreasnoack** [comment]
Maybe even mention explicitly that they are typically constructed with `sparse` and not `SparseMatrixCSC` directly.

**Comment by abraunst** [comment]
> Maybe even mention explicitly that they are typically constructed with `sparse` and not `SparseMatrixCSC` directly.

Something like this?

```
Matrix type for storing sparse matrices in the [Compressed Sparse Column](@ref man-csc) format.
The standard way of constructing SparseMatrixCSC is through the [`sparse`](@ref) function. See
also [`spzeros`](@ref), [`sprand`](@ref).
```

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 1]
```diff
@@ -10,6 +10,9 @@
 
 Matrix type for storing sparse matrices in the
 [Compressed Sparse Column](@ref man-csc) format.
+The standard way of constructing SparseMatrixCSC
+is through the [`sparse`](@ref) function. See
+also [`spzeros`](@ref), [`sprand`](@ref).
```
Comment (stevengj) [inline stdlib/SparseArrays/src/sparsematrix.jl]: Replace comma with “and”

**Review by stevengj — APPROVED** [review]

File: stdlib/SparseArrays/src/sparsematrix.jl [hunk 2]
```diff
@@ -10,6 +10,9 @@
 
 Matrix type for storing sparse matrices in the
 [Compressed Sparse Column](@ref man-csc) format.
+The standard way of constructing SparseMatrixCSC
+is through the [`sparse`](@ref) function. See
+also [`spzeros`](@ref) and [`sprand`](@ref).
```
Comment (stevengj) [inline stdlib/SparseArrays/src/sparsematrix.jl]: maybe also mention `spdiagm`

Comment (abraunst) [reply stdlib/SparseArrays/src/sparsematrix.jl]: sure, why not.

**Review by StefanKarpinski — APPROVED** [review]

**Pull request merged**